vercel.json

```diff
@@ -657,27 +653,11 @@
       "source": "/guides/deployment/deploying-to-netlify",
       "destination": "/docs/guides/deployment/deployment-guides/deploying-to-netlify"
     },
-
-    {
-      "source": "/guides/general-guides/database-workflows/cascading-deletes/:any*",
-      "destination": "/docs/guides/database/advanced-database-tasks/cascading-deletes/:any*"
```

Comment:
These two URLs should probably redirect to our Referential Actions page now instead.

Reply:
Added in these redirects